src/backoffice/shared/components/index.ts

```diff
@@ -30,3 +30,6 @@ import './input-document-picker/input-document-picker.element';
 
 import './empty-state/empty-state.element';
 import './color-picker/color-picker.element';
+
+import './debug/debug.element';
+import './debug/debug.modal.element';
```

Comment:
Consider importing the `debug.modal.element.ts` lazily inside `debug.element.ts` only when the component has to run in modal mode.

Reply:
@iOvergaard I have moved it into the main `debug.element` but I am unsure how to lazy load it and couldn't find an example in the codebase. Can you give me a pointer please.